vsts/windowsLinuxAndAndroidBuildMatrixConfig.yaml

```diff
@@ -16,26 +16,34 @@ jobs:
   ### Windows ###
   - job: Windows
     timeoutInMinutes: 180
+    strategy:
+      maxParallel: 1
+      matrix:
+        JDK 11: # always run Java 11
+          JAVA_VERSION: 11
+        JDK 8: # only run Java 8 in nightly or CI builds.
+          JAVA_VERSION: 8
     pool:
       vmImage: windows-latest
     displayName: Windows
-    condition: succeeded()
     steps:
       - powershell: ./vsts/echo_versions.ps1
+        condition: or(eq(variables['JAVA_VERSION'], 11) , ne(variables['Build.Reason'], 'PullRequest'))
```

Comment:
I couldn't find a better way to skip Java 8 if it is a pull request since this is part of a matrix strategy. A bit frustrating to have copy-pasted this condition multiple places, so if anyone knows a better way, please let me know.

Reply:
I don't like the approach we did in .NET since it involves copying the entire job and just changing the matrix values to the pri 2 targets. I'll see if your second suggestion can be used here though

Reply:
Defining the check for PR or Java 11 as a variable does work, so I'll go with that approach since there is no copy pasting of jobs or of conditionals

Reply:
Or maybe that won't work either :/